client-v2/src/components/Clipboard.tsx

```diff
@@ -80,25 +78,6 @@ class Clipboard extends React.Component<ClipboardProps> {
     this.props.dispatch(insertArticleFragmentFromDropEvent(e, to, 'clipboard'));
   };
 
-  public removeCollectionItem = (id: string) => {
```

Comment:
This is nice to remove -- we repeat ourselves in the Front and Clipboard components, and in any case this logic is probably better off living in a reducer, as it's business-y.